docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -17,86 +16,130 @@
    openvino_docs_MO_DG_prepare_model_Model_Optimizer_FAQ
 
 
-Model Optimizer is a cross-platform command-line tool that facilitates the transition between training and deployment environments, 
-performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
+To convert a model to IR, you can use the following command:
 
+.. tab-set::
 
-To use it, you need a pre-trained deep learning model in one of the supported formats: 
-TensorFlow, PyTorch, ONNX, TensorFlow Lite, and PaddlePaddle (OpenVINO support for Apache MXNet, Caffe, and Kaldi is currently 
-being deprecated and will be removed entirely in the future). Model Optimizer converts the model to the OpenVINO Intermediate Representation format (IR), 
-which you can infer later with :doc:`OpenVINO™ Runtime <openvino_docs_OV_UG_OV_Runtime_User_Guide>`.
+    .. tab-item:: mo CLI tool
+       :sync: cli-tool
 
-Note that Model Optimizer does not infer models.
+       .. code-block:: sh
 
-The figure below illustrates the typical workflow for deploying a trained deep learning model:
+          mo --input_model INPUT_MODEL
 
-.. image:: _static/images/BASIC_FLOW_MO_simplified.svg
+    .. tab-item:: mo Python API
+       :sync: mo-python-api
 
-where IR is a pair of files describing the model:
+       .. code-block:: python
 
-* ``.xml`` - Describes the network topology.
+          from openvino.tools.mo import convert_model
+          ov_model = convert_model(INPUT_MODEL)
 
-* ``.bin`` - Contains the weights and biases binary data.
+       .. note::
 
-The OpenVINO IR can be additionally optimized for inference by :doc:`Post-training optimization <pot_introduction>` that applies post-training quantization methods.
+          The ``input_model`` argument is optional. The first unnamed argument will be used as ``input_model``, therefore, both ``convert_model(input_model=mode)`` and ``convert_model(model)`` will have the same effect.
```

Comment:
Where is it optional? This is not true for CLI. In Python example there is no mentioning any `input_model` argument. @rkazants, @popovaan, @andrei-kochin you said that this document is ready. I can make a conclusion that it is not.